cmd/boostd/index.go

```diff
@@ -66,13 +62,46 @@ var indexProvListMultihashesCmd = &cli.Command{
 		}
 		defer closer()
 
-		// get list of multihashes
-		mhs, err := napi.BoostIndexerListMultihashes(ctx, propCid)
+		if cctx.Args().Len() != 1 {
```

Comment:
what's wrong with a for-loop?

Reply:
The output can be really long like upto ~55 million multihashes. It would be ideal to do this one by one. This is a very rarely used cmd. Mostly used for debugging.